blocks/autocompleters/index.js

```diff
@@ -107,7 +107,11 @@ export function blockAutocompleter( { onReplace } ) {
  * @return {Completer} Completer object used by the Autocomplete component.
  */
 export function userAutocompleter() {
-	const getOptions = () => {
+	const getOptions = ( search ) => {
+		let payload;
+		if ( search ) {
+			payload = { data: { search } };
```

Comment:
This is never used? Cause of your lint error.

Reply:
Ah, that was lost in a merge, fixed and verified its working again.